<launch>

  <node pkg="exploration_frontier_2d" type="exploration" name="exploration_frontier_2d">
    <param name="sensor_range" type="double" value="6"></param> 
    <param name="sensor_span" type="double" value="270"></param> 
    <param name="cost_function_modifier" type="double" value="1"></param> 
    <param name="cost_lambda" type="double" value="1"></param> 
    <param name="goal_function_modifier" type="double" value="1"></param> 
    <param name="goal_wait" type="bool" value="true"></param> 
  </node> 

  <node pkg="position_datalogger" type="position" name="position_datalogger"> </node> 

</launch> 
